pkg/scheduler/api/types.go

```diff
@@ -66,6 +67,16 @@ const (
 	ResizeActionUpdate        PodResourcesResizeAction = "UpdatePodForResizing"
 	ResizeActionReschedule    PodResourcesResizeAction = "DeletePodForResizing"
 	ResizeActionNonePerPolicy PodResourcesResizeAction = "PodNotResizedDueToPolicy"
+	ResizeActionUpdateDone    PodResourcesResizeAction = "UpdatePodResizingDone"
+)
+
+const (
+	// Pod not rescheduled to resize resources due to InPlaceOnly policy
+	PodResourcesResizeStatusBlockedByPolicy = "PodNotResizedDueToPolicy"
+	// Pod resources resizing request failed
+	PodResourcesResizeStatusFailed          = "PodResourceResizeFailed"
```

Comment:
nit: the wording is a little bit confusing here. it's not obvious what "failed" refers to here. try "PodResourcesStatusResizeFailed".

Reply:
this covers all failures. the podcondition generated for the failure has a reason field that carries more info.